whylog/tests/tests_log_reader/__init__.py

```diff
@@ -63,34 +69,44 @@ def _investigation_results_from_yaml(self, yaml_file, real_log_file):
         # '007_match_or',
         '008_match_and',
         # '009_match_negation',
-        # '010_multiple_files',
-        # '011_different_entry',
+        '010_multiple_files',
+        '011_different_entry',
         # '012_multiple_rulebooks',
         # '013_match_and_incomplete',
-    )
+    )  # yapf: disable
     def test_one(self, test_name):
         # paths files setup
         prefix_path = os.path.join(*TestPaths.path_test_files)
         path = os.path.join(prefix_path, test_name)
         input_path = os.path.join(path, 'input.txt')
         # output_path = os.path.join(path, 'expected_output.txt')  # FIXME is it really unnecessary?
-        log_file = os.path.join(path, 'node_1.log')
-        results_file = os.path.join(path, 'investigation_results.yaml')
+
+        original_log_file = os.path.join(path, 'node_1.log')
+        result_log_file = original_log_file
+        if test_name == "010_multiple_files":
```

Comment:
I think that current solution is good enough int current situation. Almost all tests have the same log file, except two ones. Keeping info about log files in dict would be ok, if there are many different log files.